### PR TITLE
Jf global search refactor

### PR DESCRIPTION
**Pull request by javfg**

### CODE REVIEW

File: src/public/common/search/Search.js [hunk 1]
```diff
@@ -1,113 +1,179 @@
...
+  inputBaseInput: { padding: '.25rem .5rem' },
+  root: embedded => ({ width: embedded ? '400px' : '', marginRight: '2rem' }),
+}));
 
-const IndicatorSeparator = () => null;
+function Search({ autoFocus = false, embedded = false }) {
+  const [open, setOpen] = React.useState(false);
+  const [inputValue, setInputValue] = useState('');
+  const debouncedInputValue = useDebounce(inputValue, 300);
+  const SEARCH_QUERY = loader('./SearchQuery.gql');
```
Comment (mirandaio) [inline src/public/common/search/Search.js:35]: Can you move this line outside of the component? It does not depend on props or state and we don't want it to execute for every render.

File: src/public/common/search/Option.js [hunk 2]
```diff
@@ -1,115 +1,87 @@
...
+      {data.approvedSymbol}
+    </Typography>{' '}
+    <Typography variant="caption" color="textSecondary" display="inline">
+      {data.approvedName}
+    </Typography>
+  </>
+);
+
+const Option = ({ data }) => {
+  const optionTypes = {
```
Comment (mirandaio) [inline src/public/common/search/Option.js:76]: Please move this object outside of the component. It does not depend on either props or state and we don't want to create this object on every render.

File: src/public/common/search/Group.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    fontSize: '.75rem',
+    color: theme.palette.secondary.main,
+    borderBottom: `1px solid ${theme.palette.secondary.main}`,
+  },
+}));
+
+const Group = ({ children, name }) => {
+  const classes = useStyles();
+
+  const groupName = {
```
Comment (mirandaio) [inline src/public/common/search/Group.js:20]: Please move this object outside of the component.

File: src/public/common/search/Search.js (hunk 1 above)
Comment (javfg) [reply src/public/common/search/Search.js:35]: Makes sense!

**Pull request merged**